secrets.txt

```diff
@@ -0,0 +1 @@
+Shhhhh
```

Comment:
Please note that this file should be ignored from Git. An interesting exercise would be for you to research how to remove the file from Git without deleting it from your hard drive.